samples/word/10-content-controls/insert-and-change-content-controls.yaml

```
@@ -72,14 +72,15 @@ script:
 
         async function setup() {
           await Word.run(async (context) => {
-            context.document.body.clear();
-            context.document.body.insertParagraph("One more paragraph. ", "Start");
-            context.document.body.insertParagraph("Inserting another paragraph. ", "Start");
-            context.document.body.insertParagraph(
+            let body = context.document.body;
```

Favor `const` over `let` when the variable won't be reassigned.
```suggestion
            const body = context.document.body;
```